api/v1alpha1/image_types.go

```diff
@@ -314,6 +314,12 @@ type ImageFilter struct {
 	// name specifies the name of a Glance image
 	// +optional
 	Name *OpenStackName `json:"name,omitempty"`
+
+	// tags is a list of tags which will be applied to the image. A tag has a maximum length of 255 characters.
```

Comment:
This is in filter, so the tags won't be applied.

Reply:
bad copy&paste indeed.